docs/index.md

```diff
@@ -1 +1,3 @@
-# Ansible <projectname> Documentation
\ No newline at end of file
+# Ansible PROJECTNAME Documentation
+
+<Note to developer - this should be a short paragraph about your project and why someone would want to use it>
```

Comment:
Global comment. Consider dropping the "Note to developer" thing. That just seems to add noise.